sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -300,6 +305,10 @@ def prepare_runtime(
             if sync_result.success:
                 self.files_synced = True
                 self.object_id = sync_result.object_id
+
+                for file in os.listdir("./"):
```

Comment:
shouldn't this clean up everything regardless of if the sync was successful? Also do we really need to iterate through the entire directory?

Reply:
I think there's another way to do this using tempfile: https://docs.python.org/3/library/tempfile.html